multicluster/apis/multicluster/v1alpha2/clusterset_types.go

```diff
@@ -31,13 +31,6 @@ type LeaderClusterInfo struct {
 	// Name of the Secret resource in the member cluster, which stores
 	// the token to access the leader cluster's API server.
 	Secret string `json:"secret,omitempty"`
-	// ServiceAccount in the leader cluster, from which the member cluster's token
-	// is generated. This is an optional field which helps admin to check
-	// which ServiceAccount is used by a member cluster to access the leader cluster.
-	//
-	// DEPRECATED
-	// This field is planned to be removed in the future releases.
-	ServiceAccount string `json:"serviceAccount,omitempty"`
```

Comment:
I don't know the history of this field, but should it be removed as part of a separate PR, given that this one focuses on configuration options, and this is an API-related change?

Reply:
Sure, let me create a new PR for this.

Reply:
Submitted a PR #6134 to remove the field.

Reply:
Hi @antoninbas the field removal has been fixed by the PR #6134, could you take a look at this PR again? Thanks.